packages/web-app-files/tests/unit/components/SideBar/Links/PublicLinks/LinkInfo.spec.js

```diff
@@ -0,0 +1,233 @@
+ */
+console.error = function(message) {
+  if (typeof message === 'string' && message.includes('Vue warn')) {
+    // keep default behaviour
+    // error.apply(console, arguments)
+    // Vue warn contains messages with warning initiator element
+    // Only vue warn message is thrown leaving the initiator element description
+    const errorSplit = message.split('\n')
+    throw errorSplit[0]
+  } else throw message instanceof Error ? message : new Error(message)
+}
+
+describe('LinkInfo', () => {
+  function getShallowWrapper(
+    link = {
+      token: '122235445488',
+      url: 'some-link'
+    }
+  ) {
+    return shallowMount(LinkInfo, {
+      localVue,
+      stubs: { ...stubs, 'oc-tag': true },
+      propsData: {
+        link: link
+      },
+      directives: {
+        'oc-tooltip': jest.fn()
+      }
+    })
+  }
+
+  function getMountedWrapper(
+    link = {
+      token: '122235445488',
+      url: 'some-link'
+    }
+  ) {
+    localVue.use(DesignSystem)
+    return mount(LinkInfo, {
+      localVue,
+      propsData: {
+        link: link
+      },
+      stubs,
+      directives: {
+        'oc-tooltip': jest.fn()
+      }
+    })
+  }
+
+  describe('link name', () => {
+    it('should show link name as token if link does not have name', () => {
+      const wrapper = getShallowWrapper()
+      expect(wrapper.vm.linkName).toEqual('122235445488')
+      expect(wrapper.find(selectors.linkName).text()).toEqual('122235445488')
+    })
+
+    it('should show link name if provided link has name attribute', () => {
+      const wrapper = getShallowWrapper({
+        name: 'some-name',
+        url: 'some-link'
+      })
+      expect(wrapper.vm.linkName).toEqual('some-name')
+      expect(wrapper.find(selectors.linkName).text()).toEqual('some-name')
+    })
+
+    it('should show link name if provided link has both name and token attribute', () => {
+      const wrapper = getShallowWrapper({
+        name: 'some-name',
+        token: 'some-token',
+        url: 'some-link'
+      })
+      expect(wrapper.vm.linkName).toEqual('some-name')
+      expect(wrapper.find(selectors.linkName).text()).toEqual('some-name')
+    })
+  })
+  describe('link prop', () => {
+    it('should be required', () => {
+      expect(() => {
+        shallowMount(LinkInfo, {
+          localVue,
+          stubs: { ...stubs, 'oc-tag': true }
+        })
+      }).toThrow('[Vue warn]: Missing required prop: "link"')
+    })
+    it('should not accept values other than object type', () => {
+      expect(() => {
+        shallowMount(LinkInfo, {
+          localVue,
+          stubs: { ...stubs, 'oc-tag': true },
+          propsData: {
+            link: 'some-link'
+          }
+        })
+      }).toThrow(
+        '[Vue warn]: Invalid prop: type check failed for prop "link". Expected Object, got String with value "some-link".'
+      )
+    })
+  })
+  describe('link.url', () => {
```

Comment:
to make consistent wiith above ones
```suggestion
  describe('link url', () => {
```